Sources/Stores/NetworkPersistableStore.swift

```diff
@@ -35,16 +35,15 @@ where Network.Remote == Data, Network.Request == URLRequest, Network.Response ==
 
     @discardableResult
     public func fetch<R>(resource: R, completion: @escaping NetworkStoreCompletionClosure<R.Local, E>) -> Cancelable
-    where R: NetworkResource & PersistableResource & StrategyFetchResource & RetryableResource,
-          R.Remote == Remote, R.Request == Request, R.Response == Response
-    {
+        where R: NetworkResource & PersistableResource & StrategyFetchResource & RetryableResource,
```

Comment:
nesting the `where` like this has the side effect below, of also nesting the body once more "tab", which IMO is 💩

Reply:
I do agree. Reverting as mentioned before